pydevices/acq400Devices/acq400_base.py

```diff
@@ -81,18 +81,18 @@ def init(self):
 
         # The default case is to use the trigger set by sync_role.
         if self.trig_mode.data() == 'role_default':
-            uut.s0.sync_role = "{} {}".format(self.role.data(), self.freq.data())
+            uut.s0.sync_role = "%s %s" % (self.role.data(), self.freq.data())
```

Comment:
same questions about .format

Reply:
.format does not work on rhel6.. i.e. old python2.. % is the most compatible variant